lib/chef/provider/group/pw.rb

```diff
@@ -38,7 +38,11 @@ def define_resource_requirements
 
         # Create the group
         def create_group
-          command = [ "pw", "groupadd", set_options ]
+          command = if freebsd? # Freebsd 13.x doesn't accept running the whole script under sudo and throws an error when creating/deleting groups
+                      [ "sudo", "pw", "groupadd", set_options ]
+                    else
+                      [ "pw", "groupadd", set_options ]
+                    end
```

Comment:
This seems to imply that Chef is not running as root - are you implying that Chef cannot be run as root on FreeBSD? That seems... unlikely?

Also, I don't _think_ we assume sudo is setup, generally speaking... this would add such an assumption.

Reply:
That was a dumb choice and I dunno why I thought it was. I removed it. If the tests are failing under 'sudo' we got something way bigger to solve.